src/parser-includes.ts

```diff
@@ -72,8 +80,7 @@ export class ParserIncludes {
                 const files = await globby([value["local"].replace(/^\//, ""), ...excludedGlobs], {dot: true, cwd});
                 for (const localFile of files) {
                     const content = await Parser.loadYaml(`${cwd}/${localFile}`, {inputs: value.inputs || {}});
-                    excludedGlobs.push(`!${localFile}`);
```

Comment:
@PigeonF 

After some digging, this was in introduced in https://github.com/firecow/gitlab-ci-local/pull/566#issue-1338909179, i guess the intention is to "prevent infinity check" ?  

With these new "nest include" check, shall we remove this line ?